Library/Formula/libechonest.rb

```diff
@@ -13,14 +11,14 @@ class Libechonest < Formula
     sha1 "1ae85cce12862896bcc2d28509854dd8e6b3d9be" => :mountain_lion
   end
 
-  depends_on 'cmake' => :build
-  depends_on 'qt'
-  depends_on 'qjson'
+  depends_on "cmake" => :build
+  depends_on "qt"
+  depends_on "qjson"
 
-  conflicts_with 'doxygen', :because => "cmake fails to configure build."
+  conflicts_with "doxygen", :because => "cmake fails to configure build."
 
   def install
     system "cmake", ".", *std_cmake_args
-    system "make install"
+    system "make", "install"
   end
```

Comment:
Yup, would be good if (like `tinyxml2.rb`) we could link import a header and link against the library.